app/server/src/features/utils.ts

```diff
@@ -97,7 +96,8 @@ export async function runMutation<TReturn>(cb: TCallback<TReturn>): TRunMutation
             body: result,
         };
     } catch (e) {
-        server.log.error(e.message);
+        // FIXME:
+        // server.log.error(e.message);
```

Comment:
Yeah, I'll add this back.  I removed mostly to fix the build since I moved where the server was / the import was no longer valid.

Reply:
Ah, another thing I just noticed.  We should add some guarding so that we don't always send all errors to the client here.  That could be a vulnerability.

Reply:
Yeah we should do something for that, I'm just not sure the best way to go about it. For now I've just been catching that may give too much information and re throwing them as more generic errors for the client to receive.

Reply:
We may just want to use a try/catch in all the revolvers so we can just re-throw a generic error if any error occurs in the method while still keeping the error messages relevant to request. I'm open to any other suggestions though.

Reply:
Yeah, lets make different types of errors that are thrown #255 maybe we'll bring back `ClientError` or `ServerError` -- not 100% sure.

Reply:
We can think about this more later, for now I'll just add it back since it's not super critical at the moment.

Reply:
fixed.